presto-base-jdbc/src/test/java/io/prestosql/plugin/jdbc/TestCachingJdbcClient.java

```diff
@@ -14,15 +14,176 @@
 package io.prestosql.plugin.jdbc;
 
 import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Sets;
+import io.airlift.units.Duration;
+import io.prestosql.spi.connector.ColumnMetadata;
+import io.prestosql.spi.connector.ConnectorSession;
+import io.prestosql.spi.connector.ConnectorTableMetadata;
+import io.prestosql.spi.connector.SchemaTableName;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
 import java.lang.reflect.Method;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
 import java.util.Set;
 
+import static com.google.common.collect.Sets.difference;
 import static io.prestosql.spi.testing.InterfaceTestUtils.assertAllMethodsOverridden;
+import static io.prestosql.spi.type.IntegerType.INTEGER;
+import static io.prestosql.testing.TestingSession.testSessionBuilder;
+import static java.util.Collections.emptyList;
+import static java.util.concurrent.TimeUnit.DAYS;
+import static org.assertj.core.api.Assertions.assertThat;
 
+@Test(singleThreaded = true)
 public class TestCachingJdbcClient
 {
+    private static final Duration FOREVER = Duration.succinctDuration(1, DAYS);
```

Comment:
1 day is not forever

Reply:
It is more than life time for test. Please suggest better name.

Reply:
ONE_DAY